kodi_addon_checker/__main__.py

```diff
@@ -39,15 +40,47 @@ def main():
                                      the current directory.")
     parser.add_argument("--version", action="version",
                         version="%(prog)s 0.0.1")
-    parser.add_argument("add_on", metavar="add-on", type=dir_type, nargs="*",
-                        help="optional add-on directories")
+    parser.add_argument("dir", type=dir_type, nargs="*", help="optional add-on or repo directories")
     ConfigManager.fill_cmd_args(parser)
     args = parser.parse_args()
 
-    repo_path = os.path.abspath(os.getcwd())
-    config = Config(repo_path, args)
-    ConfigManager.process_config(config)
-    check_repo(config, repo_path, args.add_on)
+    current_dir = os.path.abspath(os.getcwd())
+    if args.dir:
+        report = Report(current_dir)
+        for directory in args.dir:
+            if os.path.isfile(os.path.join(directory, "addon.xml")):
+                report.add(Record(INFORMATION, "Checking add-on %s" % directory))
+                # For add-ons try to load .tests-config.json from current directory
+                config = Config(current_dir, args)
+                addon_report = check_addon.start(os.path.abspath(directory), config)
+                report.add(addon_report)
+            else:
+                repo_path = os.path.abspath(directory)
```

Comment:
Can we print something for this case too? Just to communicate that we're using repo handling?